### PR TITLE
Add initial frontend code

### PR DESCRIPTION
**Pull request by giossa94**
@bryant1410 @pgrill I think that this first initial version of the frontend app is ready for review. Let me know if you have any questions or suggestions.

### CODE REVIEW

File: frontend/tests/__mocks__/browserMocks.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		getItem: (key) => store[key] || null,
+		setItem: (key, value) => store[key] = value.toString(),
+		clear: () => store = {}
+	};
+
+})();
+
+Object.defineProperty(window, 'localStorage', {
+	value: localStorageMock
+}); */
```
Comment (bryant1410) [inline frontend/tests/__mocks__/browserMocks.js:21]: What is this for? Can we remove it, as its commented?

**Review by pgrill — APPROVED** [review]

**Review by bryant1410 — APPROVED** [review]
LGTM! Thanks! :raised_hands: 

What are your thoughts on versioning the lock file? Mostly for reproducibility of the docker image we're gonna have for this.

I tried running it but ran into some errors when the API is being called (what exactly did you put in the `.env` file?). Gonna continue trying or get in touch soon.

File: frontend/tests/__mocks__/browserMocks.js (hunk 1 above)
Comment (giossa94) [reply frontend/tests/__mocks__/browserMocks.js:21]: @bryant1410 Yes, it could be removed, since there are no tests for now.

**Comment by giossa94** [comment]
@bryant1410 
Will open a new PR to address your feedback:
- Remove unused test utilities
- Version lock file
- Add a `.env.template` to list the environment variables needed to run the application (currently there's only one, `API_URL`, and you can set it to `/api` to run the project locally)

**Comment by bryant1410** [comment]
Cool, thanks! I set it to exactly that, but run into a CSRF token error.

**Comment by giossa94** [comment]
> Cool, thanks! I set it to exactly that, but run into a CSRF token error.

I ran into this too, and told it to @pgrill, he's going to fix it. It's an issue with the session cookies set when you log in into the admin panel (`/admin`), they seem to be sent on every request sent to the Django API, and the app endpoints are not authenticated right now.
In the meantime try deleting the cookies of localhost:8000 (or whatever port you're serving the Django API).

**Pull request merged**